### PR TITLE
Add details on Avro schema type

### PR DESCRIPTION
**Pull request by tomasz-sadura**

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 1]
```diff
@@ -73,8 +73,31 @@ If you consume JSON data with no message schema, the schema check for the connec
 
 The Avro converter requires a schema in Schema Registry.
 
+Avro supports primitive types and complex types, like records, enums, arrays, maps and unions.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
Avro supports primitive types and complex types, like records, enums, arrays, maps, and unions.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 2]
```diff
@@ -73,8 +73,31 @@ If you consume JSON data with no message schema, the schema check for the connec
...
+
+To specify timestamp in Avro schema to use with Kafka Connect, use
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
To specify a timestamp in an Avro schema for use with Kafka Connect, use:
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 3]
```diff
@@ -73,8 +73,31 @@ If you consume JSON data with no message schema, the schema check for the connec
...
+        }
+    ],
+    "default": null
+}
+----
+
 See also: xref:manage:schema-registry.adoc[Redpanda Schema Registry]
 
+See also: https://avro.apache.org/docs/1.11.1/specification[Avro specification]
+
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
See also: 

- xref:manage:schema-registry.adoc[Redpanda Schema Registry]
- https://avro.apache.org/docs/1.11.1/specification[Avro specification]

```

**Review by Feediver1 — APPROVED** [review]
Approved with updates

**Pull request merged**